bin/templates/project/dependencies.gradle

```diff
@@ -44,16 +44,14 @@ dependencies {
 	compile 'com.fasterxml.jackson.core:jackson-databind:2.5.0'
 	compile 'com.github.bumptech.glide:glide:3.6.1'
 	compile 'com.github.bumptech.glide:okhttp-integration:1.3.1'
-	compile 'com.facebook.stetho:stetho:1.3.1'
-	compile 'com.facebook.stetho:stetho-okhttp:1.3.1'
 	compile 'uk.co.chrisjenx:calligraphy:2.2.0'
 
 	// Keep in sync with root build.gradle
 	compile ('com.newrelic.agent.android:android-agent:5.6.1')
 
 	compile 'com.adobe.creativesdk.foundation:auth:0.9.978'
 
-	compile (name:'adobeMobileLibrary-4.13.0', ext:'jar') // https://github.com/Adobe-Marketing-Cloud/mobile-services
+	compile (name:'adobeMobileLibrary-4.13.2', ext:'jar') // https://github.com/Adobe-Marketing-Cloud/mobile-services
```

Comment:
You'll need to update the mobiles services jar in bin/templates/project/libs/, too.